packages/vue/src/components/__tests__/__snapshots__/federated-sign-in-button.spec.ts.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FederatedSignInButton renders as expected 1`] = `
+<div>
+  
+  <button
+    class="amplify-button amplify-field-group__control federated-sign-in-button amplify-authenticator__font"
+    data-amplify-button=""
+    data-disabled="false"
+    data-fullwidth="false"
+    data-loading="false"
+    fullwidth="false"
+    style="font-weight: normal;"
+    type="button"
+  >
+    
```

Comment:
nit: any idea why jest adds all this extra white space?

Reply:
No idea... 😓